### PR TITLE
CU-25d7ac7 - Landing page animation

### PR DESCRIPTION
**Pull request by novacooo**

### CODE REVIEW

**Comment by pmakowski23** [comment]
Task linked: [CU-25d7ac7 Landing page animation](https://app.clickup.com/t/25d7ac7)

File: components/registration/form.module.css [hunk 1]
```diff
@@ -94,9 +120,9 @@
 
 .form-row {
   position: relative;
-  max-width: 400px;
+  /* max-width: 400px; */
```
Comment (pmakowski23) [inline components/registration/form.module.css:123]: Please remove all comments

File: .env.local.example [hunk 2]
```diff
@@ -1,12 +0,0 @@
-# If you’d like to enable GitHub OAuth, fill these below
```
Comment (pmakowski23) [inline .env.local.example:1]: Please don't remove this file

**Review by pmakowski23 — CHANGES_REQUESTED** [review]
Please resolve conflicts with `master` branch

**Review by stepniaczky — APPROVED** [review]

**Review by pmakowski23 — APPROVED** [review]

**Pull request merged**